tasks/modprobeib.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: disable ipoib_enhanced
+  ansible.builtin.lineinfile:
+    path: /etc/modprobe.d/ib_ipoib.conf
+    insertafter: '^alias'
+    line: options ib_ipoib ipoib_enhanced=0
+- name: stop and start service
+  ansible.builtin.service:
+    name: openibd
+    state: stopped
+- name: start service
+  ansible.builtin.service:
+    name: openibd
+    state: started
```

Comment:
Can you use a handler instead? With `ansible.builtin.systemd` and `state: restarted`.